### PR TITLE
Secret login url

### PR DESCRIPTION
**Pull request by tspenov**
**Before merge needs env var in devops repo**

devops PR: https://github.com/santiment/devops/pull/956

GET `/bot/login/{secret_from_devops}` -> return JWT token

### CODE REVIEW

File: lib/sanbase_web/controllers/bot_login_controller.ex [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  use SanbaseWeb, :controller
+
+  require Logger
+
+  alias Sanbase.Auth.User
+
+  def index(conn, _params) do
+    user = User |> Sanbase.Repo.get_by(email: User.sanbase_bot_email())
+    {:ok, token, _claims} = SanbaseWeb.Guardian.encode_and_sign(user, %{salt: user.salt})
+    IO.inspect(token)
```
Comment (IvanIvanoff) [inline lib/sanbase_web/controllers/bot_login_controller.ex]: leftover

File: lib/sanbase_web/router.ex [hunk 2]
```diff
@@ -76,6 +80,16 @@ defmodule SanbaseWeb.Router do
     )
   end
 
+  scope "/", SanbaseWeb do
```
Comment (IvanIvanoff) [inline lib/sanbase_web/router.ex]: should this be the root or it can be something like `/bot` so we do not pollute the `/` and `/login/`

Comment (tspenov) [reply lib/sanbase_web/router.ex]: maybe `/bot` is best indeed

**Comment by IvanIvanoff** [comment]
@tspenov you can link the devops PR also here

**Pull request merged**